cli/src/build/onboarding/android/ui/app.tsx

```
@@ -0,0 +1,1798 @@
+              ;(async () => {
+                try {
+                  const bytes = await readFile(keystoreExistingPath)
+                  const base64 = bytes.toString('base64')
+                  const ready: KeystoreReady = {
+                    keystorePath: keystoreExistingPath,
+                    alias: keystoreAlias || RELEASE_ALIAS_DEFAULT,
+                    isGenerated: false,
+                  }
+                  setKeystoreBase64(base64)
+                  setKeystoreReady(ready)
+                  await persist((p) => ({
```

ESLint

[error] Unexpected parentheses around single function argument having a body with no curly braces.

(style/arrow-parens)